web/templates/_shared-base.scss

```diff
@@ -30,12 +30,7 @@ $gray-color:          #999;
 
 @font-face {
  font-family: GillSans;
- src: url(../fonts/GillSansMT.ttf);
-}
-
-@font-face {
- font-family: GillSansBold;
- src: url(../fonts/GillSansMTBold.ttf);
```

Comment:
A comment describing where this relative path leads to would be nice, eg. "Use font in web/templates/fonts"